experimental/sev_guest/src/secrets.rs

```diff
@@ -63,4 +66,21 @@ pub struct SecretsPage {
     pub tsc_factor: u32,
 }
 
+impl SecretsPage {
+    /// Gets the IMI enabled field as and `Imi` enum if possible.
+    pub fn get_imi_en(&self) -> Option<Imi> {
+        Imi::from_repr(self.imi_en)
+    }
+}
+
 static_assertions::assert_eq_size!(SecretsPage, [u8; SECRETS_PAGE_SIZE]);
+
+/// Whether an initial migration image is enabled.
+#[derive(Debug, FromRepr)]
+#[repr(u32)]
+pub enum Imi {
+    /// The initial migration image is not enabled.
+    No = 0,
+    /// The initial migration image is enabled.
+    Yes = 1,
```

Comment:
I would have called them Enabled and Disabled, but up to you.

Reply:
Done.